src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -1,3 +1,4 @@
+import { isNaN, isFinite } from 'lodash';
```

Comment:
These are already in the browser, no need to import it from lodash

Reply:
browser `isNaN(null) = false`
lodash `isNaN(null) = true`

Reply:
Can just do a null check, no need to increase bundle size by pulling in stuff from lodash.
Or just simply do typeof === 'number'